buildpack/generate.go

```diff
@@ -97,6 +102,10 @@ func runGenerateCmd(d ExtDescriptor, extOutputDir, planPath string, inputs Gener
 		EnvOutputDir+"="+extOutputDir,
 		EnvPlatformDir+"="+inputs.PlatformDir,
 	)
+	if api.MustParse(d.WithAPI).AtLeast("0.12") {
+		// todo - nil check?
+		cmd.Env = append(cmd.Env, "") // TODO platform.EnvVarsFor(&(inputs.Target))...)
+	}
```

Comment:
I don't think we need to do anything in the buildpack package, if the vars are just handed to us with the rest of the env